datafusion/physical-plan/src/filter.rs

```diff
@@ -994,4 +1014,22 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_validation_filter_selectivity() -> Result<()> {
```

Comment:
Shall we also add a test showing that changing the default selectivity actually affects the output statistics?

I think if the selectivity got hard coded to 0.2 again, no tests would fail 🤔  Maybe we could add another unit tests here setting selectivity to `0.5` or something and demonstrating the statistics are different

Reply:
I agree we should, however I didn't know how to observe the effect of such a parameter. Is there some form of observable state or result exposed that I can use to perform an assertion about what selectivity has been used ?

Reply:
@alamb fixed with https://github.com/apache/arrow-datafusion/pull/8243/commits/81034c209128b0046f464a40a5245de9c53861f5

Reply:
Not sure however how [this](https://github.com/apache/arrow-datafusion/blob/main/datafusion/physical-plan/src/filter.rs#L221) should be handled